cylc/sphinx_ext/cylc_lang/domains.py

```diff
@@ -488,3 +599,117 @@ def resolve_xref(
             contnode,
             display
         )
+
+
+# The following is a minimal domain for documenting parsec objects:
+
+def parsec_ref(tokens):
+    """The detokenise equivalent for parsec (much simpler).i
```

Comment:
```suggestion
    """The detokenise equivalent for parsec (much simpler).
```
Think this 'i' can go, unless I'm mistaken?